rust/src/core/mod.rs

```diff
@@ -20,19 +20,27 @@
 // Ordering of generic arguments
 // DI, DO, MI, MO, TI, TO, QI, QO
 
+mod domains;
+pub use domains::*;
+
+mod metrics;
+pub use metrics::*;
+
+mod measures;
+pub use measures::*;
```

Comment:
I'm not crazy about having these inside `core`. Feels like that's putting too much in there, especially if they're re-exported.

Reply:
Moved it back out!

Reply:
Thanks!